kube/src/config/mod.rs

```diff
@@ -233,6 +245,45 @@ impl Config {
     }
 }
 
+fn load_auth_header(loader: &ConfigLoader) -> Result<Authentication> {
```

Comment:
this is a much nicer here now that it's split out and dealing with the enum :+1: